ed25519jni/jvm/src/test/scala/org/zfnd/ed25519/SigningKeySeedTest.scala

```diff
@@ -0,0 +1,48 @@
+package org.zfnd.ed25519
+
+import java.nio.charset.StandardCharsets
+import java.security.SecureRandom
+import java.util.Arrays
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.must.Matchers
+import scala.collection.mutable.HashSet
+
+class SigningKeySeedTest extends AnyFlatSpec with Matchers {
+  it must "wrap SigningKeySeed (RFC 8410 - DER)" in {
+    val sksValue = BigInt("D4EE72DBF913584AD5B6D8F1F769F8AD3AFE7C28CBF1D4FBE097A88F44755842", 16)
+    val sks = SigningKeySeed.fromBytesOrThrow(sksValue.toByteArray.drop(1))
+
+    val sks_der = sks.getEncoded()
+
+    val expected_der = BigInt("302e020100300506032b657004220420d4ee72dbf913584ad5b6d8f1f769f8ad3afe7c28cbf1d4fbe097a88f44755842", 16)
```

Comment:
It may be worthwhile to switch over to reading from DER/PEM files, as is done with the main Rust tests.